src/main/java/com/github/kornilova_l/flamegraph/plugin/server/converters/file/jmc/FlightRecorderCompressedConverter.java

```diff
@@ -0,0 +1,87 @@
+package com.github.kornilova_l.flamegraph.plugin.server.converters.file.jmc;
+
+import com.github.kornilova_l.flamegraph.plugin.server.converters.file.CFlamegraph;
+import com.github.kornilova_l.flamegraph.plugin.server.converters.file.CFlamegraphLine;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+class FlightRecorderCompressedConverter extends FlightRecorderConverter {
```

Comment:
Good, just make `FlightRecorderConverter` abstract because we do not want to use it anywhere (and also remove `FlightRecorderConverter.writeTo` method)

Reply:
Why should we remove `FlightRecorderConverter.writeTo`? Won't we need this for `ParseInSeparateProcess` when the `*.jfr*` is too big? If not, we'll need to strip that code out.

Reply:
The problem is that `FlightRecorderConverter.writeTo` outputs flamegraph instead of cfamegraph. This fix requires refactoring of `CompressedFlamegraphFileSaver`, let's not worry about it now, because the PR will become too big. 
I'll create a separate branch for #48 in this repo, so we can split work in smaller chunks and I will be able to contribute